salalib/ngraph.h

```diff
@@ -105,14 +101,6 @@ class Node
    // testing some agent stuff:
    std::vector<PixelRef> m_occlusion_bins[32];
 public:
-   Node()

```

Comment:
Hmm, this might work, but the code looks hairy (not your fault, though). This `Bin m_bins[32]` member makes me nervous in the case of copy operation, though the compiler probably does the right thing...